README.md

```diff
@@ -1,9 +1,3 @@
-# Frontend 2019 p2p homework
```

Comment:
Great job! Perfectly done!

However, I would like to share my opinion. See the comments below.

Reply:
@aymkin thanks for feedback, please checks my answers and changes.